package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead",
-  "version": "1.1.3",
+  "version": "1.1.4",
```

```suggestion
  "version": "1.1.3",
```
You'll want to increment the Psammead Caption package version, not the top-level Psammead repo version.